### DISCUSSION TITLE
apm是否支持elasticsearch-apm

### DISCUSSION THREAD

**Discussion by Phoebus888** [apache/apisix#8445] [Q&A]
### Description

apm是否支持elasticsearch-apm

**Comment by tzssangglass** [comment]
I think you're asking in the wrong place, APISIX is not an APM project, but APISIX supports interfacing with elasticsearch

**Reply by Phoebus888** [reply]
插件支持  可以将tracing数据传到elasticsearch-apm  用es的apm功能展示数据

**Reply by tzssangglass** [reply]
Not supported yet. We have the elasticsearch-logger plugin, but it does not collect tracing data.
